content/en/blog/_posts/2022-05-25-contextual-logging/index.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: blog
+title: "Contextual Logging in Kubernetes 1.24"
+date: 2022-05-25
+slug: contextual-logging
+---
```

Comment:
```suggestion
canonicalUrl: https://kubernetes.dev/blog/2022/05/25/contextual-logging/
```

Reply:
We should set this, if that is the canonical URL of the other article.

Reply:
I have updated this PR to match the other one and added this canonical URL.